packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -17,6 +19,160 @@
 GLOBAL_PORT_POOL = iter(range(10000, 32000))
 
 
+class Container(enum.Enum):
+    POD = 'POD'
+
+
+class MarathonApp():
```

Comment:
`Minor`: This is bad python style. We should have had it as `class MarathonApp:`